client/reader/_style.scss

```diff
@@ -1,3 +1,19 @@
+.is-group-reader-refresh .layout__content {
+ 	padding-top: 0;
```

Comment:
Appears this has unintended consequences on recommendations view:

![screen shot 2016-09-26 at 8 27 21 pm](https://cloud.githubusercontent.com/assets/1779930/18856333/f05cd846-8427-11e6-9939-016141910450.png)

(Note lack of any padding above "We think you'll like", which differs from production)